packages/table/components/Table.tsx

```diff
@@ -122,6 +138,47 @@ export class Table<T> extends React.PureComponent<TableProps, TableState> {
     );
   }
 
+  private fillTableWidth(
+    colWidths: number[],
+    remainingWidth: number,
+    width: number
+  ) {
+    let filledWidth;
+    const totalWidth = colWidths.reduce((acc, curr) => acc + curr, 0);
+    const widthToFill = width - totalWidth;
+    const columns = React.Children.toArray(this.props.children) as Array<
```

Comment:
Let's pass `children` https://github.com/dcos-labs/ui-kit/pull/282/files#diff-f82f764378c02f64dad7e8c8b1910116R63 as an argument to the function, so that we could extract the function from the class and unit-test it.